### ISSUE TITLE
[Accessibility]: File path: Unusable on small screens

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35854]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34418

### Problem description

<img width="432" alt="image" src="https://user-images.githubusercontent.com/9516420/169800822-6bedecc3-4ddc-4df7-9a9d-179d507cdafd.png">

<img width="743" alt="image" src="https://user-images.githubusercontent.com/9516420/169801093-9a98b759-05eb-4242-97f6-7cfd08199267.png">

It is impossible to view or action the file path on small screens

### Expected behavior

We should break this onto new lines.

### Additional details

Related issue: https://github.com/sourcegraph/sourcegraph/issues/35239

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Duplicate of https://github.com/sourcegraph/sourcegraph/issues/35239, closing to avoid duplicating work!
